pydo/manager.py

```diff
@@ -330,6 +361,113 @@ def add(
             task_attributes['title'],
         )
 
+    def modify(
+        self,
+        id,
+        title=None,
+        agile=None,
+        body=None,
+        due=None,
+        estimate=None,
+        fun=None,
+        state=None,
+        priority=None,
+        project_id=None,
+        tags=[],
+        tags_rm=[],
+        value=None,
+        willpower=None,
+    ):
+        """
+        Use parent method to modify an existing task
+
+        Arguments:
+            agile (str): Task agile state.
+            title (str): Title of the task.
+            body (str): Description of the task.
+            estimate (float): Estimate size of the task.
+            fun (int): Fun size of the task.
+            state (str): State of the task once it's closed
+            priority (int): Task priority.
+            project_id (str): Project id.
+            tags (list): List of tag ids.
+            tags_rm (list): List of tag ids to remove.
+            title (str): Title of the task.
+            value (int): Objective/Bussiness value of the task.
+            willpower (int): Willpower consumption of the task.
+        """
+        if len(id) < 10:
```

Comment:
As it's used in `_close` maybe it's good to refactor this snippet to it's own method.

Reply:
Maybe inside `_update`?